Jasmine/Tetris/ViewModels/TetrisViewModel.swift

```diff
@@ -1,3 +1,14 @@
+import Foundation
+
 class TetrisViewModel: TimedGameViewModel {
+    /// Specifies the total time allowed in the game.
+    var totalTimeAllowed: TimeInterval
+
+    /// Specifies the time remaining in the game.
+    var timeRemaining: TimeInterval
 
+    init() {
+        totalTimeAllowed = 0
+        timeRemaining = 0
+    }
```

Comment:
To be honest I don't get what's wrong with this. If you have a protocol without default implementations you need to do this too right, which is to implement everything?